RELEASING.md

```diff
@@ -138,8 +138,56 @@ The first step is to ensure the GitHub release is not marked "latest," and the h
 ### Manually Promoting Downstreams
 
 If downstream promotion failed for any reason, e.g., a check failure on the same Git revision blocked promotion, then it
-is best to create a new release that fixes the problem. Manually promoting downstreams is hypothetically possible, has
-never been attempted, is error prone and tedious, and should probably be avoided.
+is probably best to create a new release that fixes the problem. Manually promoting downstreams is possible, but error
+prone and tedious.
+
+1. In GitHub, find "latest." This is the highest version that's not a pre-release, and should be available in the Linux repos.
+1. In Artifactory, explore the available non-tunneler packages. They're organized together because they are OS
+   version-neutral, while the tunneler packages are organized separately by OS version. DEB and RPM repos have distinct
+   layouts, but these links alone will answer "Is the latest stable CLI available?".
+   - [the `debian` tree](https://packages.openziti.org/zitipax-openziti-deb-stable/pool/openziti/amd64/)
+   - [the `redhat` tree](https://packages.openziti.org/zitipax-openziti-rpm-stable/redhat/x86_64/)
+
+#### RedHat Packages
+
+```bash
+(set -euxo pipefail
+# curl -sSf https://api.github.com/repos/openziti/ziti/releases/latest | jq -r '.tag_name'
```

Comment:
could always just grab this and ask "do you want to use this version"?

Reply:
Added a note about modifying to suit because I want to keep these example scripts extremely simple